exercise-10/src/com/nayema/Product.java

```diff
@@ -0,0 +1,14 @@
+package com.nayema;
+
+import java.util.Scanner;
+
+public class Product {
+    double price;
+    final double tax = 1.15; //15% sales tax
+
+    public void taxAmount() {
```

Comment:
- Pull out the setting of `price` to `Main`.
- This method should be returning the calculated taxed amount. Not setting the `price` on the `Product`.